### PR TITLE
fix(DB/Creature): Add spirit healer to gy 1318

### PR DESCRIPTION
**Pull request by Kitzunu**
* https://github.com/TrinityCore/TrinityCore/commit/acb3999bb8bc289461e9fccad04e3e4a6fd2fddb

Co-Authored-By: ModoX <moardox@gmail.com>

## Changes Proposed:
-  Add a spirit healer to gy 1318

## Tests Performed:
- Tested in-game

## How to Test the Changes:

1. .go grave 1318
2. die

## Target Branch(es):
- [x] Master

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Review by FrancescoBorzi — APPROVED** [review]

**Review by Helias — APPROVED** [review]

File: data/sql/updates/pending_db_world/rev_1611937625874584000.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1611937625874584000');
+
+DELETE FROM `creature` WHERE `guid` = 15 AND `id` = 6491;
+INSERT INTO `creature` (`guid`, `id`, `map`, `spawnMask`, `phaseMask`, `modelid`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `MovementType`) VALUES
+(15, 6491, 571, 1, 1, 0, 0, 2002.7415771484375, 5938.0986328125, 37.18986129760742187, 5.934119224548339843, 300, 0, 0);
```
Comment (BarbzYHOOL) [inline data/sql/updates/pending_db_world/rev_1611937625874584000.sql:5]: lol 5.934119224548339843

5.93 or 5.934 would work as well

Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1611937625874584000.sql:5]: Sniffed values my friend

Comment (BarbzYHOOL) [reply data/sql/updates/pending_db_world/rev_1611937625874584000.sql:5]: tell me if you see a difference with 5.934 IG

Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1611937625874584000.sql:5]: sure, but what does exact position hurt us x)

Perhaps there won't be a noticeable difference ingame, but cool to have the correct values at least =)

**Comment by pklloveyou** [comment]
Tested

**Pull request merged**